src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -254,7 +254,13 @@ Item {
                         removeMember(namesModel.get(namesList.count-1).publicId);
                         namesModel.remove((namesList.count-1), 1);
                     }
+                    if ((event.key === Qt.Key_Return || event.key === Qt.Key_Enter) && (sortedList.count > 0)) {
+                        root.insertTag(sortedList.get(userListView.currentIndex).name, sortedList.get(userListView.currentIndex).publicId);
+                    }
                 }
+                Keys.onUpPressed: { userListView.decrementCurrentIndex(); }
+                Keys.onDownPressed: { userListView.incrementCurrentIndex(); }
+                Component.onCompleted: { forceActiveFocus(); }
```

Comment:
Just one thought here: Do we really want this component to force active focus? Shouldn't the application decide which controls will need active focus in their views?

Reply:
Not really, and it's not specified either. I added it because I found it annoying to have to click on it every time in order to type, since the only point that initiates action here is the text edit, I thought adding it. But I could also remove it :) wdyt?

Reply:
done, removed from the component and added in the instance where it's used :)